routes/admins/admin_codes.js

```diff
@@ -0,0 +1,70 @@
+const crypto = require('crypto');
+
+const express = require('express');
+const { pool } = require('../../server/db');
+// const { keysToCamel } = require('../utils');
+
+const adminCodeRouter = express();
+
+adminCodeRouter.get('/', async (req, res) => {
+  try {
+    const {
+      cookies: { userId },
+    } = req;
+    const user = await pool.query('SELECT DISTINCT code FROM admin_codes WHERE user_id = $1', [
+      userId,
+    ]);
+    res.status(200).send(user.rows);
+  } catch (err) {
+    res.status(400).send(err.message);
+  }
+});
+
+adminCodeRouter.get('/code/:adminCode', async (req, res) => {
+  try {
+    const { adminCode } = req.params;
+    const user = await pool.query('SELECT * FROM admin_codes WHERE admin_codes.code = $1', [

```

Comment:
you can just do `codes = $1` for the condition if you're fetching from one table only